dlme_airflow/tasks/transform_validation.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import requests
+
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+from airflow.utils.task_group import TaskGroup
+
+
+def _fetch_transform_output(collection) -> str:
+    data_path = collection.data_path().replace(
+        "/", "-"
+    )  # penn/egyptian-museum => penn-egyptian-museum
+    transform_output_url = f"https://s3-us-west-2.amazonaws.com/dlme-metadata-dev/output/output-{data_path}.ndjson"
+    output_file_path = (
+        f"/tmp/output-{collection.provider.name}-{collection.name}.ndjson"
+    )
+    r = requests.get(transform_output_url, allow_redirects=True)
+    open(output_file_path, "wb").write(r.content)
+    return output_file_path
+
+
+def _get_transformed_record_count(collection) -> int:
```

Comment:
There is a bug in transform that is duplicating records. For this to pass until that is fixed, the output of this count should be cut in half.